front/src/applications/operationalStudies/components/Project/PictureUploader.tsx

```diff
@@ -156,7 +181,7 @@ function PicturePlaceholderButtons({ setTempProjectImage }: PropsButtons) {
 
 export default function PictureUploader({ image, setTempProjectImage, tempProjectImage }: Props) {
   const [isValid, setIsValid] = useState<boolean>(true);
-
+  const { safeWord } = useSelector((state: RootState) => state.main);
```

Comment:
please use specific selector: const safeWord = useSelector((state: RootState) => state.main. safeWord);

you will avoid unecessary rerenders